lib/main.js

```diff
@@ -1,38 +1,57 @@
 // stake is in pence
 // £10.00 becomes 1000
 
-// odds are a decimal
+// odds are decimal
 // "8/1" becomes 9.0
 
 // terms are decimal
 // "1/5" becomes 0.2
 
-// single(stake, odds, terms) {
+import { combinations } from './combinations.js';
 
-  module.exports = (function() {
-    function bet(stake, odds) {
-      return (stake * (odds - 1)) + stake;
-    }
+function bet(stake, odds) {
+  return (stake * (odds - 1)) + stake;
+}
 
-    function single(stake, odds, terms, placeStake = 0) {
-      const newPlaceStake = terms && placeStake === 0 ? stake : placeStake;
+function single(stake, odds, terms, ew, placeStake = 0) {
```

Comment:
what type is `ew` and can it be anything?. 

Typescript would be nice 🤔  (Just saying)

Also I was wondering if explicitly using json key value pairs will help anyone know what is expected for this `function`?